Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -45,8 +45,11 @@
 "action-saved-accessibility" = "Saved. Activate to unsave.";
 "action-share" = "Share";
 "action-unsave" = "Unsave";
+"adjust-text-sizing" = "Adjust text sizing";
```

Comment:
We should keep the prefix for strings in similar contexts. It's nice to have the extra info, and keeps them ordered together on the TWN side.